module/modmanager/manager_test.go

```diff
@@ -532,12 +532,6 @@ func TestModManagerValidation(t *testing.T) {
 func TestModuleReloading(t *testing.T) {
 	ctx := context.Background()
 
-	// Lower global timeout early to avoid race with actual restart code.
-	defer func(oriOrigVal time.Duration) {
-		oueRestartInterval = oriOrigVal
-	}(oueRestartInterval)
-	oueRestartInterval = 10 * time.Millisecond
```

Comment:
Does _not_ lowering this interval make the test quite long? Or, now that we don't have the three restart attempts, is this interval irrelevant?